partner_daytopay/views/res_partner.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <record id="view_res_partner_form_dtp" model="ir.ui.view">
+        <field name="name">res.partner.form.dtp</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="account.view_partner_property_form"/>
+        <field name="arch" type="xml">
+            <data>
+                <field name="property_payment_term_id" position="after">
+                    <field name="d2p_ytd"/>
+                    <field name="d2p_life"/>
+                </field>
+
+                <field name="property_supplier_payment_term_id" position="after">
+                    <field name="d2r_ytd"/>
+                    <field name="d2r_life"/>
+                </field>
+
+                <field name="credit" position="replace">
+                    <field name="credit" modifiers="{'readonly': true}"/>
```

Comment:
Why do you need this? This is a function field without inverse right and therefore already readonly, right? Maybe we could have a comment to explain why in the code?